docs/resources/tensorflow_frontend.md

```diff
@@ -13,7 +13,7 @@ Also, the frontend allows loading TensorFlow models in SavedModel, MetaGraph, an
 The current limitations:
 
 * IRs generated by new TensorFlow Frontend are compatible only with OpenVINO API 2.0
-* There is no full parity between the legacy frontend and the new frontend in MO. Known limitations compared to the legacy approach are: TF1 Control flow, Complex types, models requiring config files and old python extensions. The solution detects unsupported functionalities and provides fallback.
+* There is no full parity between the legacy frontend and the new frontend in MO. Known limitations compared to the legacy approach are: TF1 Control flow, Complex types, models requiring config files and old python extensions. The solution detects unsupported functionalities and provides fallback. To force the use of the legacy frontend, ``--use_legacy_frontend`` must be specified.
 
 To force the use of the legacy frontend, ``--use_legacy_frontend`` must be specified.
```

Comment:
You repeat ` To force the use of the legacy frontend, ``--use_legacy_frontend`` must be specified.` twice. Please check.